datacontract/model/data_contract_specification.py

```diff
@@ -147,6 +147,7 @@ class Field(pyd.BaseModel):
     format: str = None
     required: bool = None
     primary: bool = None
```

Comment:
"primary" wird schon als Feld für den Primary-Key verwendet, daher finde ich es unpassend, ein weiteres Feld einzuführen. Entweder "primary" an allen Stellen umbenennen oder das neue Attribut "primaryKey" wieder entfernen.

Reply:
`primary` is deprecated.